Source/WebKit/Configurations/WebKit.xcconfig

```diff
@@ -224,10 +224,10 @@ EXCLUDED_MACOS_PLUGIN_FILE_NAMES[sdk=iphone*] = SecItemShim.dylib WebProcessShim
 INSTALLHDRS_SCRIPT_PHASE = YES;
 APPLY_RULES_IN_COPY_HEADERS = $(WK_USE_NEW_BUILD_SYSTEM);
 
-WK_FRAMEWORK_HEADER_POSTPROCESSING_DISABLED[sdk=macosx*] = $(WK_FRAMEWORK_HEADER_POSTPROCESSING_DISABLED$(WK_MACOS_1300));
```

Comment:
Should check with Alexey -- I think there's a reason we can't do this yet.

Reply:
@aproskuryakov — should we hold off on disabling header postprocessing until later seeds?

(If so, I'll split this out into a separate change, along with the xcconfig target conditional support).

Reply:
I also have a vague feeling that matches Tim's, but can't remember any reason. Sorry.